### PR TITLE
refactor: simplify conditionals

### PR DESCRIPTION
**Pull request by blackxored**
Simplify conditionals across the code base. Includes, but not limited, to:

* Direct assignment instead of branch assignment.
* Return boolean expressions directly.
* No unnecessary `else` after return.

### CODE REVIEW

File: app/components/obs/inputs/ObsSystemFontSelector.vue.ts [hunk 1]
```diff
@@ -162,11 +162,7 @@ export default class ObsSystemFontSelector extends ObsInput<IObsInput<IObsFont>>
 
   get selectedFont() {
     return _.find(this.selectedFamily.fonts, font => {
-      if (this.value.value.flags !== this.getFlagsFromFont(font)) {
-        return false;
-      }
-
-      return true;
+      return this.value.value.flags === this.getFlagsFromFont(font);
```
Comment (gettinToasty) [inline app/components/obs/inputs/ObsSystemFontSelector.vue.ts]: this could be simplified further using implicit return but otherwise LGTM

Comment (blackxored) [reply app/components/obs/inputs/ObsSystemFontSelector.vue.ts]: Removed braces are arrow function.

File: app/util/DragHandler.ts [hunk 2]
```diff
@@ -207,11 +207,7 @@ export class DragHandler {
       return false;
     }
 
-    if (b.offset + b.length < a.offset) {
-      return false;
-    }
-
-    return true;
+    return b.offset + b.length >= a.offset;
```
Comment (andycreeth) [inline app/util/DragHandler.ts]: Same thing as above, checks returning `false`, with a return `true` at the end.  Breaks the pattern, this isn't more readable.

File: app/services/source-filters.ts [hunk 3]
```diff
@@ -121,11 +121,7 @@ export class SourceFiltersService extends Service {
       }
 
       /* Video filters can be applied to video sources. */
-      if (source.video && filterType.video) {
-        return true;
-      }
-
-      return false;
+      return source.video && filterType.video;
```
Comment (andycreeth) [inline app/services/source-filters.ts]: Same thing.  Why break the pattern of the function?  Why not combine this entire function into 1 big conditional?  (please don't do that)

**Pull request merged**